resources/assets/js/main.js

```diff
@@ -7,3 +7,22 @@ $(function() {
         }
     });
 });
+
+$(function() {
+    'use strict';
+
+    var AMOUNT_SCROLLED = 300;
+
+    $(window).scroll(function () {
+        if ($(window).scrollTop() > AMOUNT_SCROLLED) {
+            $('#back-to-top').fadeIn('slow');
+        } else {
+            $('#back-to-top').fadeOut('slow');
+        }
+    });
+
+    $("#back-to-top").click(function () {
+        $("html, body").animate({scrollTop: 0}, "slow");
+        return false;
+    });
+});
```

Comment:
brakuje nowej linii na koncu pliku (http://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file)

ja korzystam z phpstorm i mozna ustawic, aby z automatu dodawal nowa, pusta linie. do niedawna tez nie wiedzialem o tym, ale rozne narzedzia sprawdzajace poprawnosc kodu "krzycza", ze plik nie zawiera pustej linii na koncu